docusaurus.config.js

```diff
@@ -103,31 +103,86 @@ const config = {
         },
         {
           type: "localeDropdown",
-          position: "right",
+          position: "left",
         },
         {
-          href: "https://www.suse.com/c/?s=harvester",
-          position: "right",
-          label: "Blog",
-          className: "navbar__blog",
-        },
-        {
-          href: "https://harvesterhci.io/kb",
-          position: "right",
-          label: "Knowledge Base",
-          className: "navbar__kb",
+          type: 'search',
+          position: 'left',
         },
         {
-          href: 'https://www.rancher.com',
-          label: 'Rancher Home',
-          position: 'right',
+          type: `dropdown`,
+          label: `Quick Links`,
+          position: `right`,
+          items: [
+            {
+              href: "https://harvesterhci.io/",
+              label: "Harvester Home",
+              target: '_self',
```

Comment:
What's the reason for having some of the menu items use `target: _self` while others don't? It should be consistent for all menu items.